src/migration.ts

```diff
@@ -86,12 +82,8 @@ export class Migration {
 			throw new Error('Could not find migration method: ' + method);
 		}
 
-		const result = this.options.migrations.wrap(fn).apply(migration, args);
-
-		if (!isPromise(result)) {
-			throw new Error(`Migration ${this.file} (or wrapper) didn't return a promise`);
```

Comment:
Why was this check removed?

Reply:
Turning this around - why did we need it it in the first place? `await 123` and `await Promise.resolve(123)` both resolve to `123` anyway.

To answer your question, though - removing the check allows passing in jest mocks created with `jest.fn()` directly in, without having to mock the implementation when we don't really care about it.

Reply:
I think this check was here as a safeguard against people forgetting to return their promises in the migration. But maybe you can convince me this shouldn't be our problem...